enarx-keep/Cargo.toml

```diff
@@ -8,13 +8,17 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+enarx-keep-sev-shim = { path = "../enarx-keep-sev-shim" }
 sallyport = { path = "../sallyport" }
 sgx-types = { path = "../sgx-types" }
 bounds = { path = "../bounds" }
+kvm-bindings = "0.2.0"
 kvm-ioctls = "0.5.0"
 memory = { path = "../memory" }
 mmap = { path = "../mmap" }
 structopt = "0.3"
 colorful = "0.2"
 goblin = "0.2"
 libc = "0.2"
+units = { path = "../units" }
+x86_64 = { version = "0.11.0", default-features = false, features = ["stable"] }
```

Comment:
I've generally tried to keep path vs version crates segregated. Could you follow this?

Reply:
Of course. :joy: I just try to keep things alphabetized. I'll queue this up for the next revision.